core/include/usBundleContext.h

```diff
@@ -91,10 +92,61 @@ class US_Core_EXPORT BundleContext
 
 public:
 
-  BundleContext(const BundleContext&) = delete;
-  BundleContext& operator=(const BundleContext&) = delete;
+  BundleContext();
```

Comment:
Since the default ctor constructs an invalid ```BundleContext``` a comment similar to the one for ```Bundle``` would be helpful to users.